### PR TITLE
Pass the verbose flag along from the runner command to the git scm

### PR DESCRIPTION
**Pull request by orta**
I don't have the answer to https://github.com/danger/danger/issues/185 - but I think this should help in debugging it.

### CODE REVIEW

File: lib/danger/environment_manager.rb [hunk 1]
```diff
@@ -3,9 +3,11 @@
 
 module Danger
   class EnvironmentManager
-    attr_accessor :ci_source, :request_source, :scm
+    attr_accessor :ci_source, :request_source, :scm, :verbose
+
+    def initialize(env, verbose:verbose = false)
```
Comment (segiddins) [inline lib/danger/environment_manager.rb]: What's with the verbose after the `:`?

Comment (orta) [inline lib/danger/environment_manager.rb]: named parameters, e.g. `e = Danger::EnvironmentManager.new(env, verbose: true)` writing `(env, true)` doesn't really give much context when reading

Comment (orta) [inline lib/danger/environment_manager.rb]: or can you do them without that declaration?

Comment (segiddins) [inline lib/danger/environment_manager.rb]: Should be `def initialize(env, verbose: false)`

**Comment by orta** [comment]
Closing, this can be re-built sometime.

**Pull request closed without merging**